Gruntfile.js

```diff
@@ -774,9 +790,9 @@ module.exports = function(grunt) {
     'env:general',
   ]);
 
-  grunt.registerTask('test', 'Lint, unit tests, api_integration tests and e2e tests', [
+  grunt.registerTask('test', 'Lint, unit tests, integration tests and e2e tests', [
     'unit',
-    'test_api_integration',
+    'integration',
```

Comment:
Are you sure this should be replacing the api integration tests, rather than being run in addition?

Reply:
`test_api_integration` is now part of the `integration` task 👍